README.md

```diff
@@ -316,6 +321,13 @@ The migration file is a normal Bash script file:
 - It should be idempotent as much as possible to ensure that the changes are
   not duplicated to the pipeline when run the migration multiple times.
 - Pass the `shellcheck` without customizing the default rules.
+- Check whether the migration is for all kinds of Konflux pipelines or not. If
+  no, skip the pipeline properly in the script, e.g. skip FBC pipeline due
```

Comment:
Would be helpful to describe with an example how to skip FBC (but probably not here, maybe just leave a note like "see below").

Reply:
Removed this item. This one should be over-thought. An example is the pr 2018, fbc-bundle does not have task sast-coverity-check, then there will not sast-coverity-check upgrade.

Reply:
I think this would be quite a common case, we need to have it in the docs somewhere.

Reply:
Added it back.